tox.ini

```diff
@@ -21,27 +21,24 @@ deps =
     pyxdg
     jellyfish
     python-mpd2
-commands = nosetests {posargs}
-
-[testenv:py27cov]
-basepython = python2.7
-deps =
-    {[testenv]deps}
     coverage
-commands = nosetests --show-skipped --with-coverage {posargs}
-
-[testenv:py27setup]
-basepython = python2.7
-commands = python ./setup.py test {posargs}
-
-[testenv:docs]
-changedir = docs
-deps = sphinx
-commands = sphinx-build -W -q -b html . {envtmpdir}/html
 
-[testenv:flake8]
+[_flake8]
 deps =
     flake8
     flake8-future-import
     pep8-naming
-commands = flake8 beets beetsplug beet test setup.py docs
+files = beets beetsplug beet test setup.py docs
+
+[testenv]
+deps =
+    test: {[_test]deps}
+    flake8: {[_flake8]deps}
+commands =
+    test: nosetests {posargs}
+    flake8: flake8 {posargs:--min-version=2.7} {[_flake8]files}
```

Comment:
Got it; thanks. I ask because it would be good for manual `tox` executions to do the right thing automatically, rather than depending on extra arguments that the Travis config knows about. But it's no big deal for now if we don't care about the min version until the Python 3 era.